x/evm/txs/deliver/deliver.go

```diff
@@ -49,13 +49,15 @@ func (tx *Tx) GetSenderAccount() authexported.Account {
 	return pm.AccountKeeper.GetAccount(infCtx, tx.StateTransition.Sender.Bytes())
 }
 
+// ResetWatcher when panic reset watcher
 func (tx *Tx) ResetWatcher(account authexported.Account) {
 	// delete account which is already in Watcher.batch
 	if account != nil && tx.Ctx.GetWatcher().Enabled() {
 		tx.Ctx.GetWatcher().DeleteAccount(account)
 	}
 }
 
+// RefundFeesWatcher fix account balance in watcher with refund fees
 func (tx *Tx) RefundFeesWatcher(account authexported.Account, ethereumTx *types.MsgEthereumTx) {
```

Comment:
```suggestion
func (tx *Tx) refundFeesWatcher(account authexported.Account, ethereumTx *types.MsgEthereumTx) {
```
we would not export this function now

Reply:
yes, I changed the function name